routing/road_graph.cpp

```diff
@@ -112,7 +107,7 @@ bool Edge::SameRoadSegmentAndDirection(Edge const & r) const
 bool Edge::operator==(Edge const & r) const
 {
   return m_featureId == r.m_featureId && m_forward == r.m_forward &&
-         m_partOfReal == r.m_partOfReal && m_segId == r.m_segId &&
+         m_type == r.m_type && m_segId == r.m_segId &&
```

Comment:
m_type is initialized first, why not to compare by m_type first?

Reply:
Done.